### ISSUE TITLE
how to inject user root to kali image run standalone?

### ISSUE THREAD

**Issue by mjakubowskilkr** [kasmtech/workspaces-core-images#12]
I need to add user root to Kali container. Can I add any parameter to command `sudo docker run --rm  -it --shm-size=512m -p 6901:6901 -e VNC_PW=password kasmweb/<image>:<tag>` ?

**Comment by j-travis** [comment]
Yes, you are free to add any docker run flag you want , including `--user root`
https://docs.docker.com/engine/reference/commandline/run/

**Comment by mjakubowskilkr** [comment]
OK, thank you. Another question - how to secure connection, can I change username and password for authentication? Default is kasm_user / password.

**Comment by mjakubowskilkr** [comment]
Can anybody answer me?

**Comment by mmcclaskey** [comment]
For the greatest level of security, use the full Kasm Workspaces platform, which is does all the following and more
- Central authentication and user management using LDAP, SAML, or Kasm's built in authentication
- Group based security policies
- Centralized logging
- Optional two-factor authentication

For using a kasm docker image directly, without Workspaces, you can change the password as documented in Docker Hub and in GitHub in the example docker run command. The username is hard coded I believe, if you run the container directly without using the Workspaces platform.

**Comment by mjakubowskilkr** [comment]
Ok, thanx , it's satisfied answer.
